app/controllers/items/detail.js

```diff
@@ -434,8 +434,15 @@ export default GoodcityController.extend(
         this.set("selectedDescriptionLanguage", langauge);
       },
 
-      async updateAttribute(name, value, cb = _.noop) {
+      /**
+       * @param {String} name | Name of the field to update
+       * @param {any} value | Value of the field
+       * @param {boolean} isRequired | false by default
+       * @param {function} cb | Callback to invoke
+       */
+      async updateAttribute(name, value, isRequired = false, cb = _.noop) {
         const item = this.get("item");
+        if (!value && isRequired) return;
```

Comment:
A bit weird, this would prevent setting a value to `null` or `""`

Reply:
It will just prevent if value is not present and the field is required
In case value needs to be null or "" then isRequired will be false by default, so it would not affect that

Reply:
I see, I would suggest `allowEmpty` instead of `isRequired` but no biggie